packages/control-input/src/sass/_module.scss

```diff
@@ -34,16 +34,22 @@
 	cursor: pointer;
 
 	.au-control-input--small & {
-		@include AU-space( left, -0.2unit );
-		@include AU-space( top, -0.1unit );
+		left: 0;
```

Comment:
@alex-page how come we're not using `AU-space` anymore?

Reply:
When the value is 0 you don't need the function

Reply:
👍